test/test-applications/integrations/dependency-libs/TestLibrary.InstrumentationTarget/Command.cs

```diff
@@ -14,12 +14,13 @@
 // limitations under the License.
 // </copyright>
 
-using System.Threading;
+using System.Runtime.CompilerServices;
 
 namespace TestLibrary.InstrumentationTarget;
 
 public class Command
 {
+    [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
For now, in-lined methods cannot be instrumented from plugins (nor from the `OpenTelemetry.AutoInstrumentation` directly (checked with this instrumentation).

Issue occurs between df78d34cb99e9e0ab171ce241c7459cd70cc3a11 and 00ae3164889f830ce6ad2386aedc6a20e7d1a82c

Creating separate issue for that.